### ISSUE TITLE
YogaKit wont build on case sensitive systems

### ISSUE THREAD

**Issue by farfromrefug** [facebook/yoga#1035]
# Report

- [ x] I have searched [existing issues](https://github.com/facebook/yoga/issues) and this is not a duplicate

# Issues and Steps to Reproduce
Clone on a case sensitive macos system
`pod install` in the ios tutorial folder.
try to build.

# Expected Behavior
build

# Actual Behavior
wont build

The reason is that when installing pod headers for Yoga we have mismatching casing: 
<img width="563" alt="Screenshot 2020-10-23 at 16 31 09" src="https://user-images.githubusercontent.com/655344/97016553-29a0c800-154d-11eb-8112-01fd6355a423.png">

**Comment by fondue-tech** [comment]
Still issue as of 2022

**Comment by 1seck** [comment]
Still issue here.
Even in plain "react-native init test" project.
That is really bad.

**Comment by stuckj** [comment]
Yep, still broken. You can fix it by manually renaming the directory to `Yoga` and updating any references in the project. But, it's a pain and really shouldn't be necessary.
